lib/core/src/display/shape/text/text_field.rs

```diff
@@ -135,12 +135,35 @@ impl TextField {
     }
 
     /// Make change in text content.
+    ///
+    /// As an opposite to `edit` function, here we don't care about cursors, just do the change
+    /// described in `TextChange` structure.
     pub fn make_change(&mut self, change:TextChange, fonts:&mut FontRegistry) {
```

Comment:
I would name it `modify`. What do you think about it?

Reply:
It's more like `apply_change`